src/ssh.ts

```diff
@@ -0,0 +1,49 @@
+import { readFile } from "fs";
+import * as vscode from "vscode";
+const SSHConfig = require("ssh-config");
```

Comment:
```suggestion
import { readFile } from 'fs';
import * as vscode from 'vscode';
import * as sshConfig from 'ssh-config';
```

Reply:
import * as sshConfig from 'ssh-config'; requires @types/ssh-config which i didn't find in npm or a file .d.ts with a declaration should i added it?

Reply:
Oh OK, leave it as is for now then